src/screens/SavedEventListScreen/NoSavedEvents.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import React, { PureComponent } from "react";
+import { Image, StyleSheet, View } from "react-native";
+import type { NavigationScreenProp, NavigationState } from "react-navigation";
+import Text from "../../components/Text";
+import ContentPadding from "../../components/ContentPadding";
+import ButtonPrimary from "../../components/ButtonPrimary";
+import whiteCheck from "../../../assets/images/whiteCheck.png";
+import iconSave from "../../../assets/images/save.png";
+import noSavedEvents from "../../../assets/images/howToSaveEvents.png";
+import text from "../../constants/text";
+import { cardBgColor, titleTextColor } from "../../constants/colors";
+import { EVENT_LIST } from "../../constants/routes";
+import strings from "../../constants/strings";
+
+type Props = {
+  navigation: NavigationScreenProp<NavigationState>
+};
+
+class NoSavedEvents extends PureComponent<Props> {
+  eventList = () => {
+    this.props.navigation.navigate(EVENT_LIST, {
+      title: strings.featuredEventsTitle
```

Comment:
I don't think the title is needed? Also `featuredEventsTitle` is not the correct title for the event list screen 😉